packages/app/src/vis-packs/VisBoundary.tsx

```diff
@@ -23,7 +23,7 @@ function VisBoundary(props: Props) {
       fallbackRender={(args) => (
         <ErrorFallback className={visualizerStyles.vis} {...args} />
       )}
-      onError={() => valuesStore.evictCancelled()}
+      onError={() => valuesStore.evictErrors()}
```

Comment:
Note that I now evict _all_ errored requests, not just the ones that were aborted, which means that requests that errored for other reasons will be retried the next time `valueStore.prefetch()` or `valueStore.get()` is called with the same inputs. This is a great unexpected benefit from this refactoring -- until now, users had to reload the app for errors to be cleared from the fetch cache. 🎉